<launch>
  <!-- TODO: Parameterize for different bag file paths -->
  <node pkg="rosbag" type="play" name="rosbag" required="true" args="$(find head_based_intent_prediction)/recordings/bags/190617/test_2019-07-09-14-03-05.bag"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
    <remap from="image" to="/usb_cam/image_raw"/>
    <!-- TODO: recorded at 30Hz, but 1/0.03 = 33.3333Hz seems to get all the frames?-->
    <!-- if you say 0.33333 it seems to generate less images -->
    <param name="sec_per_frame" value="0.03"/>
  </node>
</launch>